src/components/Dashboard/Dashboard.module.scss

```diff
@@ -1,42 +1,56 @@
-$base-unit: 4px;
-$base-font-size: 0.3rem;
 $color-blue: #041187;
 $color-red: #e30062;
 $color-green: #85da6b;
-$color-disabled: #e5e7eb;
+$color-disabled: #7c7e82;
 $color-white: #fff;
 $color-black: #000;
-$diff-margin: $base-unit * 15;
-$similar-margin: $base-unit * 8;
-$section-padding-top-bottom: $base-unit * 30;
-$section-padding-left-right: $base-unit * 25;
 
 .container {
-    margin: $diff-margin auto;
-    width: 85%;
+    margin: 40px auto;
+    width: 95%;
+    display: flex;
+    justify-content: center;
+    line-height: 2rem;
+    align-items: center;
 }
 
-.searchLabel {
-    font-size: $base-font-size * 5;
-    padding-right: $base-unit * 8;
-    color: $color-blue;
+.searchBar {
+    width: 600px;
+    font-family: sans-serif;
+    margin-right: 8px;
+    border-radius: 4px;
+    font-size: 1.2rem;
+    padding: 0.5rem;
 }
 
-.searchBar {
-    background-color: $color-disabled;
-    border-radius: $base-unit * 2;
+.btnActive {
+    background-color: $color-blue;
+    border-radius: 4px;
+    color: white;
+    background-color: $color-blue;
     border: none;
-    width: 80%;
-    height: $base-unit * 15;
-    font-size: $base-font-size * 5;
+    padding: 12px 60px;
+    cursor: pointer;
 }
 
-.searchBtn {
-    margin-left: $base-unit * 8;
+.btnDisabled {
```

Comment:
Let's use `:disabled` selector for styling disabled button

Reply:
Thanks for the suggestion,
Implemented it please re-review